lib/Support/OSCompatWindows.cpp

```diff
@@ -407,6 +422,18 @@ int sched_getcpu() {
   return -1;
 }
 
+uint64_t cpu_cycle_counter() {
+#if defined(_MSC_VER) && (defined(_M_IX86) || defined(_M_X64))
+  return __rdtsc();
```

Comment:
Totally optional, but if you ever wanted ARM32 to build:

```suggestion
  return __rdtsc();
#elif defined(_MSC_VER) && defined(_M_ARM)
  return __rdpmccntr64();
```

Reply:
Thanks for the suggestion! We do not compile this code for ARM, so I am hesitant to include it in this PR. We can do it in future if we ever need to target ARM (not ARM64).